v2/models/notification.go

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+// Notification and its properties are defined in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/support-notifications/2.x#/Notification
+// Model fields are same as the DTOs documented by this swagger. Exceptions, if any, are noted below.
+type Notification struct {
+	Timestamps
+	Category    Category
```

Comment:
Why is type `Category` and not `NotificationCategory` like `NotificationSeverity` & `NotificationStatus`

Reply:
Renamed type `Category` to `NotificationCategory`.